ow_plexil/src/plexil-adapter/OwAdapter.cpp

```diff
@@ -32,61 +32,32 @@ using std::unique_ptr;
 
 //////////////////////// PLEXIL Lookup Support //////////////////////////////
 
-static void stubbed_lookup (const string& name, const string& value)
-{
-  // This warning is annoying.  Could parameterize it.
-  //  ROS_WARN("PLEXIL Adapter: Stubbed lookup of %s returning %s",
-  //           name.c_str(), value.c_str());
-}
-
-
-// NOTE: This macro, and the stub it implements, are temporary.
-#define STATE_STUB(name,val)                    \
-  if (state_name == #name) {                    \
-    stubbed_lookup (#name, #val);               \
-    value_out = val;                            \
-  }
-
 static bool lookup (const string& state_name,
                     const vector<PLEXIL::Value>& args,
                     PLEXIL::Value& value_out)
 {
   bool retval = true;
 
-  // Stubbed mission and system parameters.  Many of these will eventually be
-  // obsolete.
-
-  STATE_STUB(TrenchLength, 10)
-  else STATE_STUB(TrenchGroundPosition, -0.155)
-  else STATE_STUB(TrenchWidth, 10)
-  else STATE_STUB(TrenchDepth, 2)
-  else STATE_STUB(TrenchPitch, 0)
-  else STATE_STUB(TrenchYaw, 0)
-  else STATE_STUB(TrenchStartX, 5)
-  else STATE_STUB(TrenchStartY, 10)
-  else STATE_STUB(TrenchStartZ, 0)
-  else STATE_STUB(TrenchDumpX, 0)
-  else STATE_STUB(TrenchDumpY, 0)
-  else STATE_STUB(TrenchDumpZ, 5)
-  else STATE_STUB(TrenchIdentified, true)
-  else STATE_STUB(TrenchTargetTimeout, 60)
-  else STATE_STUB(ExcavationTimeout, 10)
-  else STATE_STUB(ExcavationTimeout, 60)
-  else STATE_STUB(SampleGood, true)
-  else STATE_STUB(CollectAndTransferTimeout, 10)
+  // First, some, stubbed lookups specific to EuropaMission.plp.
 
+  if (state_name == "TrenchIdentified") {
+    value_out = true;
+  }
+  else if (state_name == "ExcavationTimeout") {
+    value_out = 10;
+  }
+  else if (state_name == "CollectAndTransferTimeout") {
+    value_out = 10;
+  }
```

Comment:
Should this generate output saying this value is stubbed and therefore is simply a constant?

Reply:
Output could be repeated verbosely, and is not that useful, since it supports only one notional illustrative plan that is documented to be highly stubbed.